src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Is there a benefit to using `AkkaMaterializerSpec` rather than [uk.gov.hmrc.play.test.UnitSpec](https://github.com/hmrc/hmrctest/blob/master/src/main/scala/uk/gov/hmrc/play/test/UnitSpec.scala)?